azure/cargo-check.yml

```diff
@@ -29,14 +29,12 @@ jobs:
   - script: cargo check --locked
     condition: and(succeeded(), eq(variables.is_locked, 'true'))
     displayName: Check that Cargo.lock is satisfiable
-  - script: cargo check --all --bins --examples --tests
+  - script: cargo check --workspace --bins --examples --tests
```

Comment:
As djc mentioned, we support pinning to older Rust versions, particularly for MSRV validation.  I think the simpler approach is for us to keep using `--all` until either they plan to remove it or until the Rust version is so old we don't care about supporting it still.